modules/matlab_runtime.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+
+# Installs a MATLAB runtime when the environment variable
+# MATLAB_RUNTIME is set a runtime download URL (as a zip file).
+#
+# e.g. for MATLAB 2020a
+# MATLAB_RUNTIME=https://ssd.mathworks.com/supportfiles/downloads/R2020b/Release/8/deployment_files/installer/complete/glnxa64/MATLAB_Runtime_R2020b_Update_8_glnxa64.zip
+#
+# These links can be found here:
+# https://uk.mathworks.com/products/compiler/matlab-runtime.html
+
+set -eu
+
+name=$(echo "${MATLAB_RUNTIME}" | awk -F'/' '{ print $NF }')
```

Comment:
I've always wondered whether it was possible to get the last awk field 😄

Reply:
I love awk :3